llvm/docs/ReleaseNotes.md

```diff
@@ -240,6 +240,9 @@ Changes to Sanitizers
 Other Changes
 -------------
 
+* The -mcmodel=tiny option for the x86 architecture now triggers a frontend diagnostic.
```

Comment:
Oops, this belongs in `clang/docs/ReleaseNotes.rst` rather than in LLVM (because this is changing diagnostic behavior of the Clang driver).